jenkins/release-workflows/release-tag.jenkinsfile

```diff
@@ -58,7 +58,7 @@ pipeline {
             steps {
                 echo 'Triggering manifest lock workflow'
                 build job: 'release-manifest-commit-lock', wait: true, parameters: [
-                    string(name: 'RELEASE_VERSION', value: "${params.RELEASE_VERSION}"),
+                    string(name: 'RELEASE_VERSION', value: "${params.VERSION}"),
```

Comment:
How we lock the manifest to tag in this triggered job? We have a `product` parameters in this tag creation job but seems like we locked both OS & OSD manifests with tags in one job.

Reply:
We are in discussion to merge both OS and OSD tag creation into one run.

Reply:
Modified the workflow to accommodate both OS and OSD tag creation. In this way manifest lock would also happen once and for all.
Thanks!